packages/core/src/modules/oob/repository/OutOfBandRecord.ts

```diff
@@ -56,17 +60,22 @@ export class OutOfBandRecord extends BaseRecord<DefaultOutOfBandRecordTags> {
       this.reusable = props.reusable ?? false
       this.mediatorId = props.mediatorId
       this.reuseConnectionId = props.reuseConnectionId
-      this._tags = props.tags ?? {}
+      this._tags = props.tags ?? {
+        recipientKeyFingerprints: props.outOfBandInvitation
+          .getInlineServices()
+          .map((s) => s.recipientKeys)
+          .reduce((acc, curr) => [...acc, ...curr], [])
+          .map((didKey) => DidKey.fromDid(didKey).key.fingerprint),
+      }
```

Comment:
I'm not sure we want to set this here. It doesn't take the public dids into account and will only be set if no custom tags were provided. I.e. If I create the record like this:

```ts
const record = new OutOfBandRecord({
  anotherTag: '10'
})
```

It won't set the fingerprints , which I think is not desired.

Reply:
The resolution of recipient key fingerprints moved out of the constructor, and now the values are passed on each spot a new OOB record is created.